webpack.config.js

```diff
@@ -0,0 +1,68 @@
+import path from 'path';
+import webpack from 'webpack';
+import UglifyJSPlugin from 'uglifyjs-webpack-plugin';
+
+import {settings} from './gulp/config';
+
+const dev = {
+	entry: {
+		polyfills: './src/app/polyfills.js',
+		app: './src/app/index.js',
+		vendor: settings.sources.externalJs
+	},
+	output: {
+		path: path.join(__dirname, settings.destinations.dev.app),
+		filename: '[name].bundle.js'
+	},
+	plugins: [
+		new webpack.optimize.CommonsChunkPlugin({
+			name: ['app', 'vendor', 'polyfills']
```

Comment:
I wonder why we do have a `polyfills` bundle. Doesn’t it make sense to bundle the polyfills with the other vendor packages?